lang/programming.talon

```diff
@@ -44,9 +44,7 @@ toggle funk: user.code_toggle_functions()
 toggle library: user.code_toggle_libraries()
 funk <user.code_functions>:
     user.code_insert_function(code_functions, "")
-library <user.code_libraries>:
-    insert("library()")
-    key(left)
+(include|import|library) <user.code_libraries>:
```

Comment:
I agree, seems like this should be language specific. Include doesnt make sense in python, import doesnt in c, etc.

Reply:
i moved it to per file